src/js_native_api_v8.cc

```diff
@@ -1718,6 +1718,32 @@ napi_status NAPI_CDECL node_api_create_property_key_utf16(napi_env env,
   });
 }
 
+napi_status NAPI_CDECL napi_set_named_property_len(napi_env env,
+                                                   napi_value object,
+                                                   const char* utf8name,
+                                                   size_t name_length,
+                                                   napi_value value) {
+  std::u16string utf16name;
```

Comment:
Please use `CHECK_NEW_FROM_UTF8_LEN` instead of raw casting.

Reply:
thank you for suggestions, ı applied your comments